commands/validate.md

```
@@ -0,0 +1,24 @@
+Validate skill compliance using the compliance validator.
+
+**Usage:**
+- `/ring:validate <skill-name>` - Check compliance for skill
+- `/ring:validate <skill-name> <rule-id>` - Check specific rule
+
+**Example:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)